doc/source/whatsnew/v1.1.0.rst

```diff
@@ -75,7 +75,9 @@ Deprecations
 ~~~~~~~~~~~~
 - Lookups on a :class:`Series` with a single-item list containing a slice (e.g. ``ser[[slice(0, 4)]]``) are deprecated, will raise in a future version.  Either convert the list to tuple, or pass the slice directly instead (:issue:`31333`)
 - :meth:`DataFrame.mean` and :meth:`DataFrame.median` with ``numeric_only=None`` will include datetime64 and datetime64tz columns in a future version (:issue:`29941`)
--
+- :meth:`util.testing.assert_almost_equal` now accepts both relative and absolute
+  precision through the ``rtol``, and ``atol`` parameters, thus deprecating the
+  ``check_less_precision`` parameter. (:issue:`13357`).
```

Comment:
check_less_precision -> check_less_precise?

Is this actually deprecated, or just removed?

Reply:
I added the deprecation warning, as I understood that it was the advised process to follow. Should it just be removed instead?